### PR TITLE
some minor changes to your script, see commit history for details

### PR DESCRIPTION
**Pull request by Dulus0**
some minor changes to your script, see commit history for details

### CODE REVIEW

File: timelapse.py [hunk 1]
```diff
@@ -67,36 +67,45 @@ def capture_image():
         set_camera_options(camera)
 
         # Capture a picture.
-        camera.capture(dir + '/image{0:05d}.jpg'.format(image_number))
+        camera.capture(dir + '/image{0:08d}.jpg'.format(image_number+1))
```
Comment (geerlingguy) [inline timelapse.py]: The one change I think I'm not in favor of is starting at 1... though someone may be able to make a good argument for it. I like 0-indexed, as that makes dealing with the series easier later.

Comment (Dulus0) [reply timelapse.py]: ok, works for me, i just started to mess around timelapses, so probably there is a good reason it starts from 0.

File: timelapse.py [hunk 2]
```diff
@@ -67,36 +67,46 @@ def capture_image():
         set_camera_options(camera)
 
         # Capture a picture.
-        camera.capture(dir + '/image{0:05d}.jpg'.format(image_number))
+        camera.capture(dir + '/image{0:08d}.jpg'.format(image_number))
+        sleep(1)
```
Comment (geerlingguy) [inline timelapse.py]: What's the purpose of adding a 1 second sleep here? If you have shorter intervals, this could interfere with the timing.

**Review by geerlingguy — COMMENTED** [review]
One last question—after that I'm 100% happy with this PR, thanks!

**Comment by geerlingguy** [comment]
Thanks again for these improvements! I just tagged 2.1.0 with them included :)

**Pull request merged**